hudi-client/hudi-client-common/pom.xml

```diff
@@ -129,6 +123,11 @@
       <version>${project.version}</version>
       <classifier>tests</classifier>
       <type>test-jar</type>
+    </dependency>
```

Comment:
`hudi-hadoop-common` should still be test scoped?

Reply:
Got it.  If only tests need them, should `<scope>test</scope>` be kept?

Reply:
Yeah we are keeping `<scope>test</scope>` the diff is a bit confusing due to git